src/test/java/com/emc/mongoose/storage/driver/kafka/integration/CreateSingleTopicTest.java

```diff
@@ -0,0 +1,36 @@
+package com.emc.mongoose.storage.driver.kafka.integration;
+
+import com.emc.mongoose.storage.driver.kafka.util.docker.KafkaNodeContainer;
+import org.apache.kafka.clients.admin.*;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Properties;
+
+public class CreateSingleTopicTest {
+
+  private static KafkaNodeContainer KAFKA_NODE_CONTAINER;
+  private static AdminClient adminClient;
+  private static Properties properties = new Properties();
+
+  @Test
+  public void createTopic() throws Exception {
+    adminClient.createTopics(Collections.singleton(new NewTopic("test-topic", 2, (short) 1)));
```

Comment:
this call returns the result which should be tested accordingly

Reply:
Done